bootloader/ota-dfu-python/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
This is probably an unnecessary change.

Reply:
My Python installation generates these cache files when I run the DFU scripts in `bootloader/ota-dfu-python`. 

Personally I don't participial care for this change, I thought it could be useful. Should I remove it?

Reply:
I see. It does generate this folder for me too. Let's leave it then.